sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -123,7 +123,7 @@ object ScalaNativePluginInternal {
     nativeLinkerReporter in NativeTest := (nativeLinkerReporter in Test).value,
     nativeOptimizerReporter := tools.OptimizerReporter.empty,
     nativeOptimizerReporter in NativeTest := (nativeOptimizerReporter in Test).value,
-    nativeGC := "boehm",
+    nativeGC := Option(System.getenv.get("SCALANATIVE_GC")).getOrElse("boehm"),
```

Comment:
Lets also add `SCALANATIVE_MODE` similarly to `SCALANATIVE_GC`.

Reply:
Done.